src/stylesheets/components/_site-search.scss

```diff
@@ -78,7 +78,7 @@ $icon-size: 40px;
     width: 100%;
     margin-left: 0;
     outline: 3px solid $govuk-focus-colour;
-    outline-offset: 0;
+    outline-offset: -2px;
```

Comment:
@dashouse do you think we should make these changes as part of the focus state work? Wonder if it'd make sense to look at it separately alongside #937?

Reply:
Just the line?